### PR TITLE
gpl: modify log for routability and timing-driven modes

### PR DESCRIPTION
**Pull request by gudeh**
This PR introduces no changes in functionality.
It includes the relative change in area due to routability inflation and timing-driven area adjustments from repair_design modifications.
This information is important for addressing current challenges in gpl related to divergences.

### CODE REVIEW

File: src/gpl/src/routeBase.cpp [hunk 1]
```diff
@@ -675,11 +675,14 @@ std::pair<bool, bool> RouteBase::routability()
...
-             "InflatedAreaDelta:",
-             block->dbuAreaToMicrons(inflatedAreaDelta_));
+  log_->info(
+      GPL,
+      45,
+      "{:20} {:10.3f} um^2 ({:+.2f}%)",
+      "InflatedAreaDelta:",
+      block->dbuAreaToMicrons(inflatedAreaDelta_),
+      (static_cast<float>(inflatedAreaDelta_) / nbVec_[0]->nesterovInstsArea())
+          * 100.0f);
```
Comment (eder-matheus) [inline src/gpl/src/routeBase.cpp:685]: This block is very confuse. Please refactor lines 684 and 685 to a variable with meaningful name.

Also, it seems that the `InflatedAreaDelta` string is an argument to the fmt string below? Maybe simplify it to `"InflatedAreaDelta: {:10.3f} um^2 ({:+.2f}%)",`

File: src/gpl/src/nesterovPlace.cpp [hunk 2]
```diff
@@ -479,8 +479,12 @@ int NesterovPlace::doNesterovPlace(int start_iter)
...
           log_->info(GPL,
                      107,
-                     "Timing-driven: RSZ delta area:     {}",
-                     block->dbuAreaToMicrons(nbc_->getDeltaArea()));
+                     "Timing-driven: repair_design (rsz) delta area:     "
+                     "{:.3f} um^2 ({:+.2f}%)",
+                     block->dbuAreaToMicrons(nbc_->getDeltaArea()),
+                     (nbc_->getDeltaArea()
+                      / static_cast<float>(nesterov->nesterovInstsArea()))
+                         * 100.0f);
```
Comment (eder-matheus) [inline src/gpl/src/nesterovPlace.cpp:487]: Similar comment here, refactor these lines to a variable for clarity.

File: src/gpl/src/nesterovPlace.cpp [hunk 3]
```diff
@@ -479,8 +479,12 @@ int NesterovPlace::doNesterovPlace(int start_iter)
 
           log_->info(GPL,
                      107,
-                     "Timing-driven: RSZ delta area:     {}",
-                     block->dbuAreaToMicrons(nbc_->getDeltaArea()));
+                     "Timing-driven: repair_design (rsz) delta area:     "
```
Comment (eder-matheus) [inline src/gpl/src/nesterovPlace.cpp:482]: I think the `(rsz)` in this message is just noise. Just `"Timing-driven: repair_design delta area: "` should be enough.

Also, why the long whitespace here?

**Review by eder-matheus — CHANGES_REQUESTED** [review]

File: src/gpl/src/routeBase.cpp (hunk 1 above)
Comment (gudeh) [reply src/gpl/src/routeBase.cpp:685]: This format `"{:20} {:10.3f} um^2 ({:+.2f}%)",` is easier to align with other messages, for example:

image

File: src/gpl/src/nesterovPlace.cpp (hunk 3 above)
Comment (gudeh) [reply src/gpl/src/nesterovPlace.cpp:482]: no reason for the whitespace, I am gonna remove

File: src/gpl/src/routeBase.cpp (hunk 1 above)
Comment (eder-matheus) [reply src/gpl/src/routeBase.cpp:685]: Makes sense, thanks!

**Review by eder-matheus — APPROVED** [review]

**Pull request merged**